openerp/api.py

```diff
@@ -862,7 +873,10 @@ def __new__(cls, cr, uid, context):
 
         # otherwise create environment, and add it in the set
         self = object.__new__(cls)
-        self.cr, self.uid, self.context = self.args = (cr, uid, frozendict(context))
+        self.args = (cr, uid, frozendict(context))
```

Comment:
Wouldn't think so, though it shouldn't make any difference given it's a frozendict.